apps/mobile/src/components/headers/animated-header/animated-title-header.tsx

```diff
@@ -14,25 +17,45 @@ const AnimatedBox = Animated.createAnimatedComponent(Box);
 
 interface AnimatedTitleHeaderProps {
   animatedStyle: ReturnType<typeof useAnimatedStyle>;
+  animatedBlurOverlayStyle: ReturnType<typeof useAnimatedStyle>;
   title: string;
   rightElement?: ReactNode;
 }
 export function AnimatedTitleHeader({
   animatedStyle,
   title,
   rightElement,
+  animatedBlurOverlayStyle,
 }: AnimatedTitleHeaderProps) {
   const router = useRouter();
+  const { themeDerivedFromThemePreference } = useSettings();
+  const theme = useTheme<Theme>();
+  const gradientTransparentStopColor =
+    themeDerivedFromThemePreference === 'light' ? 'rgba(255,255,255,0)' : 'rgba(27,26,23,0)';
```

Comment:
Could we add a `const`  / a token for the gradient?

Reply:
I've debated doing this but gave up. We don't currently have alpha color tokens in the design system, and Figma doesn't support using color variables where alpha colors would matter most (gradients, shadow filters) making adding them kind of unjustified.
What this code should **really** look like is: `const gradientTransparentStopColor 
 = transparentize(theme.colors['ink.background-primary'])`, except pulling in a color library didn't feel justified either.
 
 Do you think we should make an issue out of this to look into alpha color use-cases?

Reply:
If you want do but it's not a big deal really 👍